lte/gateway/python/magma/pipelined/ifaces.py

```diff
@@ -31,8 +32,13 @@ def monitor_ifaces(iface_names):
         yield from asyncio.sleep(POLL_INTERVAL_SECONDS)
 
 
-def get_mac_address(interface_name: str) -> str:
-    if_addresses = netifaces.ifaddresses(interface_name)[netifaces.AF_LINK]
-    if not if_addresses or not if_addresses[0].get('addr'):
-        raise ValueError(f"No mac address found for interface {interface_name}")
-    return if_addresses[0]['addr']
+def get_mac_address(network_request=True, **kwargs) -> str:
```

Comment:
I wonder if it even makes sense to keep this wrapper instead of just using `getmac` directly. Adding a wrapper just for some extra error handling and argument validation is usually not worth it.

(If you call it directly, you can also remove the corresponding unit test.)

Reply:
We removed the wrapper